src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationEventSource.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics.Tracing;
+
+namespace System.Xml.Serialization
+{
+    [EventSource(Name = "System.Xml.Serialzation.XmlSerializationEventSource")]
+    internal class XmlSerializationEventSource : EventSource
```

Comment:
Don't include the word EventSource in the name as the fact it's an EventSource is an internal implementation detail.